python-package/securexgboost/core.py

```diff
@@ -2124,4 +2191,4 @@ def get_split_value_histogram(self, key, feature, fmap='', bins=None, as_pandas=
         if as_pandas and not PANDAS_INSTALLED:
             sys.stderr.write(
                 "Returning histogram as ndarray (as_pandas == True, but pandas is not installed).")
-        return nph
+        return nph
```

Comment:
Please revert this

Reply:
reverted i think? Added a newline at the end